doc/contributing/releases.md

```diff
@@ -944,6 +944,9 @@ a `NODEJS_RELEASE_HOST` environment variable:
 NODEJS_RELEASE_HOST=proxy.xyz ./tools/release.sh
 ```
 
+> In case <https://nodejs.org/dist/index.json>, <https://nodejs.org/dist/index.tab>
```

Comment:
```suggestion
> [!TIP]
> In case <https://nodejs.org/dist/index.json>, <https://nodejs.org/dist/index.tab>
```

If you'd like, Github supports syntax to add a little alert box:
> [!TIP]
> ...

> [!NOTE]
> ...

> [!WARNING]
> ...


> [!CAUTION]
> ...